abtesting/README.md

```diff
@@ -1,35 +1,65 @@
-A/B Testing
-========
+# Firebase A/B Testing Quickstart
 
-Firebase A/B Testing leverages Remote Config to automatically trial new app
-behaviors and exports the experiment results into Firebase Analytics. This
-sample demonstrates using an A/B test to test multiple color schemes within
-an app.
+## Firebase A/B Testing
+
+Firebase A/B Testing leverages Remote Config to automatically trial new app behaviors and exports 
+the experiment results into Firebase Analytics. This sample demonstrates using an A/B test to test 
+multiple color schemes within an app.
+
+See [OUTLINE.md](OUTLINE.md) for information on the design of the app, screenshots, and symbol 
+references.
+
+To view the older Objective-C and Swift quickstarts, view the 
+[LegacyABTestingQuickstart](LegacyABTestingQuickstart) directory.
+
+## Getting Started
+
+- [Add Firebase to your iOS Project](https://firebase.google.com/docs/ios/setup)
+- [Create Firebase Remote Config Experiment with A/B Testing](https://firebase.google.com/docs/ab-testing/abtest-config)
+- Run the sample on your iOS device or simulator.
 
 ## Create Experiment
 
-In Firebase Console's A/B Testing section, click the `Create experiment` button
-and select the `Remote Config` option. For Basics, provide a name for the experiment
-and optionally a description. For Targeting, select your app using the drop-down
-and increase the 'Exposure' to 100%. For Goals, feel free to choose any metric from
-the drop-down, such as `Crash-free users`. For Variants, click `Choose or create new`
-underneath 'Parameter', type `color_scheme`, and click the `Create parameter`
-drop-down presented. Under 'Baseline', provide a value of `light` to `color_scheme`.
-Under 'Variant A', provide a value of `dark` to `color_scheme`. Finally, press the
-`Review` button.
+In Firebase Console's A/B Testing section, click the `Create experiment` button and select the 
+`Remote Config` option. For Basics, provide a name for the experiment and optionally a description.
+ For Targeting, select your app using the drop-down and increase the 'Exposure' to 100%. For Goals,
+  feel free to choose any metric from the drop-down, such as `Crash-free users`. For Variants, 
+  click `Choose or create new` underneath 'Parameter', type `color_scheme`, and click the 
+  `Create parameter` drop-down presented. Under 'Baseline', provide a value of `light` to 
+  `color_scheme`. Under 'Variant A', provide a value of `dark` to `color_scheme`. Finally, press 
+  the `Review` button.
```

Comment:
it looks like some additional whitespace got added at the beginning of these lines

Reply:
Quoting above:
> That was intentional in that the previous character in the sentence is already at column 100, but this seeming whitespace doesn't show up in the Markdown rendering. Please let me know if you think I should tidy this up.

Reply:
Nevermind, it does appear that these are extra. Thanks for catching these!

Reply:
Fixed ✅